scripts/run-integration-tests-npm.sh

```diff
@@ -46,14 +46,17 @@ declare npm_install_dir="/tmp/hopr-npm"
 declare node1_dir="/tmp/hopr-npm-node-1"
 declare node2_dir="/tmp/hopr-npm-node-2"
 declare node3_dir="/tmp/hopr-npm-node-3"
+declare node4_dir="/tmp/hopr-npm-node-4"
 
 declare node1_log="${node1_dir}.log"
 declare node2_log="${node2_dir}.log"
 declare node3_log="${node3_dir}.log"
+declare node4_log="${node4_dir}.log"
 
 declare node1_id="${node1_dir}.id"
 declare node2_id="${node2_dir}.id"
 declare node3_id="${node3_dir}.id"
+declare node4_id="${node3_dir}.id"
```

Comment:
typo? `node3_dir` -> `node4_dir`

Reply:
Thank for pointing this out. Needs some refactoring though to remove duplication.